input/resources/au-core-requester.xml

```diff
@@ -4151,7 +4153,7 @@
 <name value="birthdate"/>
 <definition value="http://hl7.org/fhir/SearchParameter/individual-birthdate"/>
 <type value="date"/>
-<documentation value="A requester **SHALL** provide a value precise to the *day*.&#xa;&#xa;A responder **SHALL** support a value a value precise to the *day*."/>
+<documentation value="The requester **SHALL** provide a value precise to the *day*.&#xa;&#xa;The responder **SHALL** support a value a value precise to the *day*.&#xa;&#xa;The requester **SHOULD** support these search comparators `gt`, `lt`, `ge`, `le`.&#xa;&#xa;The requester **SHOULD** support `multipleAnd` and if `multipleAnd` is supported, the search comparators `gt`, `lt`, `ge`, `le` **SHALL** be supported."/>
```

Comment:
Missing changes to include search comparators & multipleAnd requirements for the responder.

Reply:
fixed